.sempoconfig/common_config.ini

```diff
@@ -1,78 +1,82 @@
 [APP]
```

Comment:
@enjeyw are these supposed to be committed? i don't see a gitignore for them

Reply:
Came in with Nick's seed data merge to master

Reply:
egh, this is why i like rebase better :/ having a hard time figuring out where philip's changes are in the commit tree to see how this happened, whereas with rebase, they'd just be at the top